clms/types/content/use_case.py

```diff
@@ -86,6 +129,61 @@ class IUseCase(model.Schema):
         default={},
     )
 
+    latitude = schema.Double(
```

Comment:
`schema.Double` does not exist, it should be `schema.Float`

Reply:
See the field reference here: https://training.plone.org/5/mastering-plone/dexterity_reference.html